Example/AccessibilitySnapshot.xcodeproj/xcshareddata/xcbaselines/607FACE41AFB9204008FA782.xcbaseline/EDA548A6-E453-4CBA-9366-15D413356833.plist

```diff
@@ -11,7 +11,7 @@
 				<key>com.apple.XCTPerformanceMetric_WallClockTime</key>
 				<dict>
 					<key>baselineAverage</key>
-					<real>17.200000</real>
+					<real>5.930000</real>
```

Comment:
Setting both the permissible missed width and height to `1` gives another 65% improvement over #177, leading to a total ~88% improvement over #119.

Reply:
Setting both to `2` brings it down to 1.961 sec.

Setting both to `4` brings it down to 1.074 sec.